packages/moonstone/Scroller/Scroller.js

```diff
@@ -312,7 +312,7 @@ class ScrollerBase extends Component {
 				pos = isVerticalDirection ? top : left,
 				max = isVerticalDirection ? maxTop : maxLeft;
 
-			if (pos > 0 && pos < max) {
+			if (pos >= 0 && pos < max) {
```

Comment:
`pos` can be `max` also. `maxTop` is `scrollHeight` - `clientHeight` and it is the maximum position. So I wonder it is possible that the issue occurs on the last item, too.